include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -192,9 +192,66 @@ class alignment_algorithm :
 
         return results;
     }
+
+    //!\overload
+    template <std::ranges::forward_range indexed_sequence_pairs_t>
```

Comment:
This is not required to be an indexed_sequence_pair_range (although the name suggests it)?